package.json

```diff
@@ -1,40 +1,57 @@
 {
   "name": "ember-introjs",
-  "version": "1.0.0",
-  "description": "The default blueprint for ember-cli addons.",
+  "version": "2.0.0",
+  "description": "Intro.js wrapper component",
+  "keywords": [
+    "ember-addon"
+  ],
+  "license": "MIT",
```

Comment:
OK, with `MIT` but we should also warn somewhere users that Intro.js is on `AGPL-3.0`